src/utils/worklets/chat/messenger/composer.cljs

```diff
@@ -6,14 +6,6 @@
   [chat-list-scroll-y composer-focused? window-height]
   (.scrollDownButtonOpacity ^js worklets chat-list-scroll-y composer-focused? window-height))
 
-(defn jump-to-button-opacity
-  [scroll-down-button-opacity-sv composer-focused?]
-  (.jumpToButtonOpacity ^js worklets scroll-down-button-opacity-sv composer-focused?))
-
-(defn jump-to-button-position
-  [scroll-down-button-opacity-sv composer-focused?]
-  (.jumpToButtonPosition ^js worklets scroll-down-button-opacity-sv composer-focused?))
-
 (defn composer-container-opacity
```

Comment:
Since you are touching this file, we might as well clean-up other unused functions if you are okay with that. `composer-container-opacity`, `blur-container-elevation`, and `composer-elevation` are unused and their JS implementation could be removed once we remove the CLJS ones.